### PR TITLE
feat: Added json file for XP Pen Deco Fun XS

### PR DESCRIPTION
**Pull request by sarsembin**
Added configuration file for XP Pen Deco Fun XS, tested with release  0.5.3.2 portable, working normally.

### CODE REVIEW

**Comment by AbstractQbit** [comment]
Config format has changed since 0.5.3.2, please follow the new format.

**Comment by X9VoiD** [comment]
Also, does it have buttons on the tablet? If yes, use the XP-Pen parser used on other tablets like G640S for example to add support to the tablet buttons.

**Comment by sarsembin** [comment]
It doesn't have buttons.

> Also, does it have buttons on the tablet? If yes, use the XP-Pen parser used on other tablets like G640S for example to add support to the tablet buttons.

It doesn't have them.

**Comment by sarsembin** [comment]
> Config format has changed since 0.5.3.2, please follow the new format.

What version do I need?

**Comment by AbstractQbit** [comment]
> What version do I need?

The one you are targeting with this PR, i.e. master. You can download dev builds from Actions tab.

**Comment by InfinityGhost** [comment]
image

Name should be changed to `XP-Pen CT430` to match the product model name.

File: OpenTabletDriver/Configurations/XP-Pen/Deco Fun XS.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "Name": "XP-Pen Deco Fun XS",
```
Comment (InfinityGhost) [inline OpenTabletDriver/Configurations/XP-Pen/Deco Fun XS.json:2]: Change the name and update the filename to match.
```suggestion
  "Name": "XP-Pen CT430",
```

**Review by InfinityGhost — CHANGES_REQUESTED** [review]
Update the `TABLETS.md` file adding this tablet, making sure that it is sorted correctly.

**Comment by sarsembin** [comment]
@InfinityGhost thank you, I wasnt able to do those changes myself since I couldnt find 0.6 build and build script doesnt work on my computer, so cant check if the cfg file would work.

**Review by X9VoiD — APPROVED** [review]

**Pull request merged**